mrmustard/lab/circuit_drawer.py

```diff
@@ -61,8 +61,7 @@ def _add_op(op, layer_str, decimals):
     control = []
     if op.__class__.__qualname__ in ["BSgate", "MZgate", "CZgate", "CXgate"]:
         control = [op.modes[0]]
-    # label = op.short_name
```

Comment:
This undoes a change that I had done accidentally, fixing the circuit_drawer tests in prototype-tn